### PR TITLE
Make a voters viewer

### PR DESCRIPTION
**Pull request by yet-another-account**
Now we can see how much each person has voted, as per the voters.json. Not pretty style-wise, but it works.

### CODE REVIEW

**Comment by yet-another-account** [comment]
Here's what it looks like right now:

**Comment by yet-another-account** [comment]
Not sure what the numbers mean, but I'm guessing that every time chaos polls a PR, it increments the number for all voters.

File: server/voters.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
```
Comment (o3LL) [inline server/voters.html]: Can we keep website without jquery ? Look like everybody can't do js without jquery :(

File: server/voters.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<center>
+    <h1>Voters on Chaos</h1>
+    <div id="result"></div>
+</center>
+
+<script>
+    console.log(getText());
```
Comment (o3LL) [inline server/voters.html]: dont forget to remove console.log()

File: server/voters.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                            'names': keys[j],
+                            'votes': values[j]
+                        });
+
+                    //2) sort:
+                    list.sort(function(a, b) {
+                        return -((a.votes < b.votes) ? -1 : ((a.votes == b.votes) ? 0 : 1));
+                    });
+
+                    console.log(list);
```
Comment (o3LL) [inline server/voters.html]: dont forget to remove console.log()

File: server/voters.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                    var tablehtml = "<table>";
```
Comment (o3LL) [inline server/voters.html]: try adding class from bulma : http://bulma.io/documentation/elements/table/

**Comment by yet-another-account** [comment]
I got rid of the `console.log`s

File: server/voters.html (hunk 1 above)
Comment (yet-another-account) [reply server/voters.html]: I'm not sure how to do it without jquery

Comment (PlasmaPower) [reply server/voters.html]: You literally never use jquery. You can remove this script tag and nothing will happen.

**Review by PlasmaPower — CHANGES_REQUESTED** [review]

**Comment by PlasmaPower** [comment]
:+1: We can do graphics updates later, but this is a good start.

**Comment by PlasmaPower** [comment]
Did chaosbot crash? Its check isn't showing up.

Edit: the website is still up though.

**Comment by o3LL** [comment]
> Did chaosbot crash? Its check isn't showing up.

Looks like.

> 👍 We can do graphics updates later, but this is a good start.

For the graphical part, I think this list should be hidden on the index page, and show up when you click on a button ?

**Comment by PlasmaPower** [comment]
@MUCHZER Yeah my thought as well. Bootstrap has modals, but we're not using it. Maybe use a separate modal library?

**Comment by o3LL** [comment]
> Yeah my thought as well. Bootstrap has modals, but we're not using it. Maybe use a separate modal library?

@PlasmaPower We are using Bulma -> http://bulma.io/documentation/components/modal/
But I think modal are not UX-friendly. Something like https://codepen.io/MUCHZER/pen/WjBvoN with animation in "Hello, Chaos!" panel.

**Comment by rudehn** [comment]
I can pull up the voters.json and view that, but not voters.html. All I'm getting there is the header. Possible bug?

**Comment by PlasmaPower** [comment]
@rudehn it works for me: http://chaosthebot.com/voters.html

**Comment by o3LL** [comment]
Any error message ?

**Comment by syvb** [comment]
When I go to it, I don't see an error, but all I see is a heading that says "Voters on Chaos". When I look at the source, the syntax highlighting everything after the < in ``` for (var j = 0; j < keys.length; j++)``` is the color of a tag.

**Comment by syvb** [comment]
Found it:
``` Uncaught exception: TypeError: 'Object.values' is not a function```

**Comment by PlasmaPower** [comment]
@Smittyvb get a new browser

**Comment by o3LL** [comment]
@Smittyvb Thanks. I will try to fix this when I got time

**Comment by PlasmaPower** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values#Browser_compatibility

Probably a good idea to change it, it's currently Chrome and Firefox only.

**Comment by o3LL** [comment]
Browser compatibility problem:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values
Use .map() instead

**Comment by o3LL** [comment]
Updated https://github.com/chaosbot/Chaos/issues/319

**Comment by mdcfe** [comment]
Reposting from [my comment on #319](https://github.com/chaosbot/Chaos/issues/319#issuecomment-304622659):

> In #354, I've switched to a monospace font for the page to make it a little bit more readable, and added a prefix to show the positions of each voter.
> 
> I'm not convinced about making it look the same as the index page or using Bulma, though - it looks quite simple as it is.

**Pull request merged**